interfaces/CMakeLists.txt

```diff
@@ -206,7 +211,7 @@ add_test(NAME python_multiphase_nonlinear_constraints
         COMMAND "${CMAKE_COMMAND}" -E chdir ${PROJECT_SOURCE_DIR}/examples/acados_python/multiphase_nonlinear_constraints
         python main.py)
 
-# qpDUNES example
+# qpDUNES example TODO couple with pendulum examples
```

Comment:
why not just add it in the list below?

Reply:
Will cause the ctest to report a failure, if the module is not found when built with ACADOS_WITH_QPDUNES=OFF